<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VDO</class>
 <widget class="QWidget" name="VDO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cert Stat </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ID Header</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Discover Identity Data Object Generator</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Product</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="output">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>486</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>530</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>20</y>
     <width>741</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>110</y>
     <width>741</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>741</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>20</width>
     <height>91</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>450</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Passive Cable/Active Cable/Alternate Mode DO (common configs)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>470</y>
     <width>197</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;VDM Header,ID Header, Cert Stat, Product ID&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="buddy">
    <cstring>output</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>510</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;Data Objects&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit_9</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear text boxes</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBox"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Number of Data Objects to generate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>192</width>
     <height>63</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <property name="verticalSpacing">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>USB comms Capable as USB Host</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>USB comms Capable as USB Device</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string>Modal Operation Supported</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>USB Product Type</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>102</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_9">
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>16 bit USB Vendor ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>570</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="horizontalSpacing">
     <number>12</number>
    </property>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_16"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Message:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>213</width>
     <height>169</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>9</number>
    </property>
    <property name="verticalSpacing">
     <number>1</number>
    </property>
    <item row="2" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Enter USB Type</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Enter HW Version</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="hw_ver"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Enter FW Version</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="fw_ver"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="cab_lat"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="cur_cap"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Cable Termination Type</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="usb_type"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>Enter Cable Latency</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>USB SuperSpeed Signaling Sprt</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="vconn_pwr">
      <property name="text">
       <string>Vconn Power (Alt Mode)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="cab_term"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="vcon_pwr"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="sup_speed"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Vbus Current Handling Cap</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>188</width>
     <height>166</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="verticalSpacing">
     <number>4</number>
    </property>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBox_8">
      <property name="text">
       <string>Vbus through cable</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="checkBox_9">
      <property name="text">
       <string>SOP* controller present (Act Cab)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_6">
      <property name="text">
       <string>SSRX1 Directionality Support</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBox_20">
      <property name="text">
       <string>Vbus required (Alt Mode)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_4">
      <property name="text">
       <string>SSTX1 Directionality Support</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox_7">
      <property name="text">
       <string>SSRX2 Directionality Support</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="checkBox_19">
      <property name="text">
       <string>Vconn required (Alt Mode)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_5">
      <property name="text">
       <string>SSTX2 Directionality Support</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>135</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>XID (32 bit unsigned int)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cert"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="spacing">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Product ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>BCD Device</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="bdc_dev"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="pd_id"/>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>label_15</zorder>
  <zorder>label_43</zorder>
  <zorder>label</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>label_26</zorder>
  <zorder>vcon_pwr</zorder>
  <zorder>label_2</zorder>
  <zorder>label_27</zorder>
  <zorder>label_19</zorder>
  <zorder>sup_speed</zorder>
  <zorder>output</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>label_5</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>layoutWidget_5</zorder>
  <zorder>lineEdit_9</zorder>
  <zorder>label_25</zorder>
  <zorder>cab_term</zorder>
  <zorder>layoutWidget_6</zorder>
  <zorder>lineEdit_16</zorder>
  <zorder>pushButton</zorder>
  <zorder>line_7</zorder>
  <zorder>cur_cap</zorder>
  <zorder>line_5</zorder>
  <zorder>line_6</zorder>
  <zorder>spinBox</zorder>
  <zorder>layoutWidget_7</zorder>
  <zorder>line_8</zorder>
  <zorder>line_9</zorder>
  <zorder>label_20</zorder>
  <zorder>vconn_pwr</zorder>
  <zorder>pd_id</zorder>
  <zorder>checkBox_19</zorder>
  <zorder>checkBox_20</zorder>
  <zorder>checkBox_3</zorder>
  <zorder>label_25</zorder>
  <zorder>cab_term</zorder>
  <zorder>cert</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_2</zorder>
 </widget>
 <tabstops>
  <tabstop>comboBox_2</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>checkBox_3</tabstop>
  <tabstop>cert</tabstop>
  <tabstop>pd_id</tabstop>
  <tabstop>bdc_dev</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>hw_ver</tabstop>
  <tabstop>fw_ver</tabstop>
  <tabstop>usb_type</tabstop>
  <tabstop>cab_lat</tabstop>
  <tabstop>cab_term</tabstop>
  <tabstop>cur_cap</tabstop>
  <tabstop>sup_speed</tabstop>
  <tabstop>vcon_pwr</tabstop>
  <tabstop>checkBox_4</tabstop>
  <tabstop>checkBox_5</tabstop>
  <tabstop>checkBox_6</tabstop>
  <tabstop>checkBox_7</tabstop>
  <tabstop>checkBox_20</tabstop>
  <tabstop>checkBox_8</tabstop>
  <tabstop>checkBox_19</tabstop>
  <tabstop>checkBox_9</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>output</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>lineEdit_16</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
